Objects/Objects/GIS/PointElement.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using Objects.Geometry;
+using Speckle.Core.Models;
+
+namespace Objects.GIS;
+
+public class PointElement : Base
+{
+  public List<Point>? geometry { get; set; }
+  public Base? attributes { get; set; }
+}
```

Comment:
This PointElement, NonGeometryElement, LineElement classes make me think what we need is a wrapper class that holds some typed geometry and attributes.

Reply:
this is addressed in a new branch, I removed all these and created just "GisElement"